### DISCUSSION TITLE
new search doesn't find pages with query == page title

### DISCUSSION THREAD

**Discussion by StefanoCecere** [squidfunk/mkdocs-material#4772] [Q&A] [answered]
hi.
i'm developing a quite big websites (hundreads of pages.. each one for a boardgame or a videogame)

https://2042ed.org/

if I seach for "zombie", it does correctly show many pages having "zombie" in their title
if I search for "santorini" it shows the main https://2042ed.org/played/boardgame/santorini/ page
but if i search for "quarto", "othello" or other.. their pages doesnt show up, but appears just the tags page

how is it possibile? they are very similar pages with the same structure

**Comment by squidfunk** [comment]
Thanks for testing the beta! Could you please try and create a minimal reproducible example that showcases the issue? I'm happy to look into it, but your repository is too big and it will take too much time to get it up and running. Thanks!

**Comment by StefanoCecere** [comment, marked as answer]
ah discovered and solved:
that page was "empty" (just the title and a automatic table generated via frontmatter data)
now i populated the page with text and images, and it appeared as first result in the search.

so that index probably doesont' like "empty pages".. like Google :)

<img width="853" alt="Screenshot 2022-12-21 at 09 03 34" src="https://user-images.githubusercontent.com/640536/208852214-0079edfc-cdec-4b3c-b98b-565edb8fbc0a.png">

**Reply by squidfunk** [reply]
Ah I see, well, if you can construct a case without your customizations and overrides that leads to the same problem, please file an issue. The search should index all pages that are known to MkDocs.

**Reply by StefanoCecere** [reply]
> Ah I see, well, if you can construct a case without your customizations and overrides that leads to the same problem, please file an issue. The search should index all pages that are known to MkDocs.

i'll do.. i have a separated repo just for dev (it's getting slow to render the big site)
